exoctk/exoctk_app/templates/phase_constraint.html

```diff
@@ -16,10 +16,16 @@ <h1>Phase Constraint Calculator</h1>
     <br>
 
     <p>
-        The Phase Constraint Calculator provides a simple interface for calculating the JWST observation start window.  
-        The calculation currently only applies to transits, though one can subtract 0.5 from the phase values to compute 
-        the eclipse observation start window for planets on circular orbits.  Enter the minimum and maximum phase values 
-        into the APT special requirements section when planning your observations. 
+        Due to uncertainties on the exact scheduling time of JWST observations, users are strongly encouraged to input minimum and maximum phases on the APT "Timing" requirements under the "Special Requirements" 
+        section when planning their observations. This gives the observatory a range of time (the "Window Size") on which observations might start. This is illustrated in the following diagram:
+    </p>
+    <br>
+    <p>
+    <img src="/static/images/phase_constraint.png" alt="Phase-constraint diagram." width="1000px">
+    </p>
+    <br>
+    <p>
+    The <b>Phase Constraint Calculator provides a simple interface for calculating this JWST observation window for direct input on APT.</b> This tool always assumes the zero-phase is at mid-transit, and thus that value is the one that should be inputted under "Zero phase (HJD)" in APT. Users can either resolve target properties from exo.MAST using the "Resolve Target" button below or input their own. If using the former, <b>users are strongly encouraged to check that the retrieved properties are up-to-date</b>.
```

Comment:
Small comment - second sentence should be "This tool always assumes the zero-phase is at mid-transit, and thus that value is the one that should be _input_ under "Zero phase (HJD)" in APT."

Reply:
Fixed in https://github.com/ExoCTK/exoctk/pull/357/commits/d469b8700aa26fd4d05fba3cbcae398423d6365b!